scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
```

Comment:
Is this the same file as the one stored under `github.com/apache/flink-docker`?

Reply:
Yes it is the same one in terms of its content. I suggest we keep the resources for releasing images here for easier testing.

Reply:
If we do, then we probably should group `docker_entrypoint.sh` and `Dockerfile` under `/docker`. But we might reinvent the wheel here since `flink-docker` already allows us to build images locally.